core/domain/topic_domain_test.py

```diff
@@ -1333,39 +1237,33 @@ def setUp(self):
             1, 1, 1, 1, 1, 1, 1, 'image.svg', '#C6DCDA', 'url-frag',
             current_time, current_time)
 
-    def _assert_validation_error(self, expected_error_substring):
+    def _assert_validation_error(  # type: ignore[override]
```

Comment:
Again, why is this ignore needed?

Reply:
same as the other one.

Reply:
Again, add a comment.

Reply:
In this file `core/domain/topic_domain.py`, we are using two methods with the same name `_assert_validation_error`, and for that, Mypy checks were giving [this](https://user-images.githubusercontent.com/89520981/148721778-6c68ba4d-f755-4fe7-9836-7130cecc2192.png) kind of errors. To silence them I have used those comments.

Reply:
Sure, but please add this info into the code.

Reply:
I added a few comments, please have a look.

Reply:
The comment is wrong though, sorry I didn't realize it before. The problem is not that the method is defined twice in this file. The problem is that the `_assert_validation_error` is defined in the base class of this class and we override its signature. So the comment should say something like we override the definition of the function from the parent class, but that is fine as `_assert_validation_error` is supposed to be customizable.

Reply:
I have updated the comment. Please have a look.